src/lib/time-formats.js

```diff
@@ -1,14 +1,34 @@
-/* exported formats */
+/* exported getLocalizedDateTime */
 
-export const formats = {
-  default: {
-    short_time: '[@] ha',
-    short_date_time: 'ddd [@] ha',
-    long_date_time: 'ddd MMM D \ [@] ha'
-  },
-  'it': {
-    short_time: '[@] H',
-    short_date_time: 'ddd [@] H',
-    long_date_time: 'ddd D MMM \ [@] H'
+export const getLocalizedDateTime = function (time, format) {
+  const uiLocale = browser.i18n.getUILanguage().replace('_', '-') || 'en-US';
+  let formattedDateTime;
+
+  switch (format) {
+    case 'date_day':
+      formattedDateTime = time.toDate().toLocaleString(uiLocale, {weekday: 'short', month: 'short', day: 'numeric'});
```

Comment:
it seems like you could extract the date from the object once at the top of the function instead of in each individual call?